velox/dwio/common/tests/utils/FilterGenerator.h

```diff
@@ -86,7 +89,7 @@ class AbstractColumnStats {
       float selectPct,
       FilterKind filterKind,
       const std::vector<RowVectorPtr>& batches,
-      const Subfield& subfield,
+      const FilterSpec& filterSpec,
```

Comment:
Same, FilterSpec already contains startPct, selectPct and filterKind. Other places too.